pkg/master/controller.go

```diff
@@ -40,14 +41,17 @@ import (
 )
 
 const kubernetesServiceName = "kubernetes"
+const clusterInfoConfigMapName = "cluster-info"
+const clusterID = "cluster-id"
```

Comment:
Are we still planning on doing this? I haven't heard anything about it in a while.